static/app/views/issueDetails/groupDistributions/tagExportDropdown.tsx

```diff
@@ -30,39 +31,41 @@ export default function TagExportDropdown({tagKey, group, organization, project}
   });
 
   return (
-    <DropdownMenu
-      size="xs"
-      trigger={triggerProps => (
-        <Button
-          {...triggerProps}
-          borderless
-          size="xs"
-          aria-label={t('Export options')}
-          icon={<IconDownload />}
-        />
-      )}
-      items={[
-        {
-          key: 'export-page',
-          label: t('Export Page to CSV'),
-          // TODO(issues): Dropdown menu doesn't support hrefs yet
-          onAction: () => {
-            window.open(
-              `/${organization.slug}/${project.slug}/issues/${group.id}/tags/${tagKey}/export/`,
-              '_blank'
-            );
+    <Feature features="organizations:discover-query" organization={organization}>
```

Comment:
`Feature` is already wrapped with `withOrganization` so we shouldn't have to pass organization in here

Reply:
okay i believe i updated it. but this will just hide the button right. do we have an example of showing an "upgrade to business" hook?

Reply:
i don't think we have a unified upgrade to business hook since each place that I know of that has this upsell, does it slightly differently